src/fontMetrics.ts

```diff
@@ -184,16 +183,16 @@ const extraCharacterMap = {
 };
 
 export type CharacterMetrics = {
-    depth: number;
-    height: number;
-    italic: number;
-    skew: number;
-    width: number;
+    depth: number,
+    height: number,
+    italic: number,
+    skew: number,
+    width: number
 };
 
 export type MetricMap = {
-    [string]: number[]
-}
+    [K in string]: number[];
```

Comment:
Something we may want to do in the future is change this type to be:
```
[K in string]: [number, number, number, number, number];
```
not necessary for this PR.